packages/@lwc/engine-core/src/framework/renderer.ts

```diff
@@ -15,7 +15,6 @@ type E = HostElement;
 export type LifecycleCallback = (elm: E) => void;
 
 export interface RendererAPI {
-    isNativeShadowDefined: boolean;
```

Comment:
Now that we are dropping Edge 18, all of our tested browsers support shadow DOM.